js/binance.js

```diff
@@ -2895,6 +2896,170 @@ module.exports = class binance extends Exchange {
         return this.parseTrades (response, market, since, limit);
     }
 
+    async editOrder (id, symbol, type, side, amount, price = undefined, params = {}) {
```

Comment:
@Dan-krm no matter what values I use for the price and amount I always get this `PERCENT_PRICE_BY_SIDE` error which is weird I think

Using LTC/USDT I've tried price = 10,20,30,40,50,60,70,etc 

![image](https://user-images.githubusercontent.com/43336371/207847039-224f7c04-2928-4fde-9781-e1f6508d7442.png)

Reply:
nvm, my fault 😅